zmon_agent/main.py

```diff
@@ -77,16 +95,19 @@ def add_new_entities(all_current_entities, existing_entities, zmon_client, dry_r
         try:
             logger.info('Found {} new entities to be added in ZMon'.format(len(new_entities)))
             for entity in new_entities:
-                logger.info(
-                    'Adding new {} entity with ID: {}'.format(entity['type'], entity['id']))
+                logger.info('Adding new {} entity with ID: {}'.format(entity['type'], entity['id']))
 
                 resp = zmon_client.add_entity(entity)
 
                 resp.raise_for_status()
         except Exception:
+            current_span.set_tag('error', True)
             logger.exception('Failed to add entity!')
+            current_span.log_kv({'exception': traceback.format_exc()})
             error_count += 1
 
+    current_span.log_kv({'new_entities': new_entities})
```

Comment:
do we really need the full entities here? isn't the id enough?

Reply:
Will remove